downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -60,6 +60,21 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 #### Language Server
 
+- Introduced the `Extract to transform function` code action to extract a value expression into a transform function.
+- Introduced the `Surround with lock` code action to wrap an isolated variable access with a `lock` statement.
+- Introduced the `Add private qualifier` code action to set the visibility qualifier of a field in an isolated class to `private`.
+- Introduced the `Make variable immutable` code action to add `final` and/or `readonly` as needed to make the field immutable.
+
+#### Ballerina Shell
+
+- Added support for invoking actions directly from the shell prompt, as shown in the following examples.
+
+```ballerina
```

Comment:
or bash/cmd?

Reply:
Used Ballerina to enable syntax highlighting, and since the same approach has been followed in previous release notes: https://github.com/ballerina-platform/ballerina-dev-website/blob/d11afb0dcdf6b4b121c4bc5440a937aaf604d8d6/downloads/swan-lake-release-notes/swan-lake-2201.0.0/RELEASE_NOTE.md?plain=1#L363-L377